extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,55 @@
+
+![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
 
-### Branch and Pull Request analysis
-Whatever type of source repository you are analysing, when a build is run on a branch of your project, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube:
+### Automatically Analyze Branches and Decorate Pull Requests (SonarQube Commercial Editions)
+ This extension allows automatic analysis of all branches and pull-requests which enables early discovery of bugs and vulnerabilities prior to a merge. Branches and PRs get their own Quality Gate status and results are pushed right to your project space. 
 
 ![Branches](img/branches.png)
 
-If you configure your build definition as a build validation for pull requests of that project (this can be done on "Branch policies"), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence:
+If you configure your build definition as a build validation for pull requests of that project (this can be done in 'Branch Policies'), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence.
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
 
 ![PR-Decoration](img/pull-request-decoration.png)
 
 **Important note**: to activate pull request decoration, you must specify a user token in the "General Settings > Pull Requests" administration page of your project in SonarQube.
 
-### Quality Gate Status
-The **Publish Quality Gate Result** task waits for the analysis report to be consumed by the SonarQube in order to flag the build job with the Quality Gate status. The Quality Gate is a major, out-of-the-box, feature of SonarQube. It provides the ability to know at each analysis whether an application passes or fails the release criteria. In other words it tells you at every analysis whether an application is ready for production "quality-wise".
+## About the SonarQube Azure DevOps Marketplace Extension
+This extension provides the following features:
+* A dedicated **SonarQube EndPoint** that defines the SonarQube server to be used.
+* Branch and **Pull Request** analysis in your projects.
+* Three build tasks to get your projects analyzed easily:
+  * **Prepare Analysis Configuration:** Configures all the required settings prior to executing a build. This task is mandatory. For .NET solutions or Java projects, this task helps SonarQube seamlessly integrate with MSBuild, Maven and Gradle tasks.
+  * **Run Code Analysis:** Executes the source code analysis. This task isn't required for Maven or Gradle projects.
+  * **Publish Quality Gate Result:** Displays the Quality Gate status in the build summary. This tasks is optional, as it may increase the overall build time.
```

Comment:
It might be nice to include screenshots of these in-use. I was mystified as to where/how to use these tasks when I first got started. 

You can see a build pipeline definition sample using them here: https://dev.azure.com/sonarsource-demo-001/My%20API/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=3

Reply:
Ok, I set up a meeting to go over all this! One thought to keep in mind though, this page is about promotion and now how to use the extension. Thanks for the help!

Reply:
Hi Jeff. I've updated the PR. Would you mind checking out the latest edits and letting me know what you think? Thanks.